README.md

```diff
@@ -16,43 +16,70 @@ Independent analyzers are executed with their module name, e.g. `python3.7 -m un
 Other Universum modes, such as poller or submitter, are called via command line, e.g.
 `python3.7 -m universum poll`
 
-## Installation from GitHub
+## Installation
 
-### Latest release
+### Latest release from PyPI
 
-Minimum prerequisites ([see documentation for details](
-https://universum.readthedocs.io/en/latest/prerequisites.html)):
+Minimum prerequisites ([see documentation for details](https://universum.readthedocs.io/en/latest/install.html)):
 1. OS Linux
-2. Python 3.7
-3. Pip for python3.7
+2. Python version 3.7 or greater
+3. Pip version 9.0 or greater
 ```bash
 sudo pip3.7 install -U universum
 ```
 or
 ```bash
 pip3.7 install --user -U universum
 ```
+Can also be installed with [extras for using VCS](
+https://universum.readthedocs.io/en/latest/install.html#vcs-related-extras),  but they also require
+installing respective command-line tools, such as git or p4.
 
 ### Latest development + tests
 
-Additional prerequisites ([see documentation for details](
-https://universum.readthedocs.io/en/latest/prerequisites.html#optional-used-for-internal-tests)):
-1. Perforce CLI, P4Python (['helix-cli' and 'perforce-p4python'](
-https://www.perforce.com/manuals/p4sag/Content/P4SAG/install.linux.packages.install.html))
-2. Docker (['docker-ce', 'docker-ce-cli'](
-https://docs.docker.com/install/linux/docker-ce/ubuntu/#install-docker-ce))
-3. Current user added to 'docker' group (use `sudo usermod -a -G docker $USER` and then relogin)
+Additional prerequisites for testing Universum locally:
+1. Support of all [VCS extras](https://universum.readthedocs.io/en/latest/install.html#vcs-related-extras),
+   including installation of Git and P4 CLI
+2. Manual installation of Docker (`docker-ce`, `docker-ce-cli`) (see [official installation manual](
+   https://docs.docker.com/engine/installation/linux/ubuntu/#install-using-the-repository) for details)
+
+   * Also add current user to 'docker' group (use `sudo usermod -a -G docker $USER` and then relogin)
 ```bash
+python3.7 -m venv virtual-environment-python3.7
+source ./virtual-environment/bin/activate
 git clone https://github.com/Samsung/Universum.git universum-working-dir
 cd universum-working-dir
 git checkout master
-pip3.7 install --user -U .[test]
+pip install -U .[test]
 make images
+make test
```

Comment:
I am not sure we should include "make test" into the main installation sequence

Reply:
Well if we refactor it into "Development", then why not include `make test` to command list

Reply:
The first part should be all about setup: venv, pip, etc.
You can add make test as a final step to verify the installation, but it shouldn't be a part of installation.
You can add it in separate block.